CHANGELOG_PENDING.md

```diff
@@ -21,6 +21,7 @@ Friendly reminder, we have a [bug bounty program](https://hackerone.com/tendermi
 - [types] [\#4417](https://github.com/tendermint/tendermint/issues/4417) VerifyCommitX() functions should return as soon as +2/3 threashold is reached.
 - [examples/kvstore] [\#4509](https://github.com/tendermint/tendermint/pull/4509) ABCI query now returns the proper height (@erikgrinaker)
 - [cmd] \#4515 Change `tendermint debug dump` sub-command archives filename's format (@melekes)
+- [privval] \#4534 Add `error` as a return value on`GetPubKey()`
```

Comment:
```suggestion
- [privval] [\#4534](https://github.com/tendermint/tendermint/pull/4534) Add `error` as a return value on`GetPubKey()`
```

Reply:
Shouldn't we link the issue here? https://github.com/tendermint/tendermint/issues/3602

Reply:
>It's also acceptable for xxx to refer to the relevent issue number, but pull-request numbers are preferred. 

https://github.com/tendermint/tendermint/blob/master/CONTRIBUTING.md#changelog